src/Entities/Order.php

```diff
@@ -57,17 +60,16 @@ class Order extends Entity
     public $currency;
     public $additionalInfo;
     public $orderComment;
-
     public $deliveryCost;
     public $subTotalVatPrice;
     public $totalVatPrice;
     public $subTotalPrice;
     public $totalPrice;
     public $totalPriceWithoutDelivery;
-
-    public $items = [];
-
     public $createDate;
     public $updateDate;
+
+    public $items;
```

Comment:
why do you remove default value as array?

Reply:
> why do you remove default value as array?
why do we need it?

Reply:
As for me it is good to have default value as empty array instead of null if we expect there array